src/components/Settings/SettingsJobsCache/index.tsx

```diff
@@ -66,7 +77,7 @@ const SettingsJobs: React.FC = () => {
     await axios.post(`/api/v1/settings/jobs/${job.id}/run`);
     addToast(
       intl.formatMessage(messages.jobstarted, {
-        jobname: job.name,
+        jobname: intl.formatMessage(messages[job.id]),
```

Comment:
We should design this to check if the message actually exists. If it doesn't exist there should be a fallback string like `Unknown Job.`